pdfminer/pdftypes.py

```diff
@@ -185,6 +186,23 @@ def stream_value(x):
     return x
 
 
+def decompress_corrupted(data):
+    d = zlib.decompressobj(zlib.MAX_WBITS | 32)
```

Comment:
What does `zlib.MAX_WBITS | 32` mean? Can you add a comment explaining why this setting is used?

Reply:
honestly this is copied/paste from the pypdf patch mentionned in the commit message. As I've no rational for this I'll rather remove this `| 32` for now.